utilities/validation.js

```diff
@@ -0,0 +1,71 @@
+const { body, param, validationResult } = require('express-validator');
+const validate = {};
+
+validate.idRule = () => {
+    return [param('id').isMongoId()];
+};
+
+validate.checkId = (req, res, next) => {
+    let errors = [];
+    errors = validationResult(req);
```

Comment:
Why use let here to assign an empty array to errors and then reassign it in the next line instead of

```suggestion
    const errors = validationResult(req);
```

Reply:
I don't understand why, but this is the methodology in the express-validator module documentation. I didn't make the module; I'm just implementing it how the documentation says to.